astropy/units/quantity.py

```diff
@@ -40,6 +40,52 @@
 _UFUNCS_FILTER_WARNINGS = {np.arcsin, np.arccos, np.arccosh, np.arctanh}
 
 
+def allclose(a, b, rtol=1.e-5, atol=None, **kwargs):
```

Comment:
This is a bit of a detail, but this function is not as important as the `Quantity` class - I would suggest moving it to the end of the file (with `_unquantify_allclose_arguments`)